src/wblib/figures/internal/_general_plotting_functions.py

```diff
@@ -0,0 +1,35 @@
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+import orcestra.sat
+
+def plot_sattrack(valid_time, ax, satellite:str = "EARTHCARE"):
+    valid_day = str(valid_time.date())
+    sattracks = orcestra.sat.SattrackLoader(satellite, "2024-07-22"
```

Comment:
This is potentially dangerous:
The function `plot_sattrack` silently adds the forecast issuing date as an (unchangable) default argument. Any particular satellite track forecast will be outdated and we should **not** do flight planning on outdated and thus possibly wrong satellite track forecasts.

Reply:
Yes, I agree. Thanks for the hint! I'll change that.

Reply:
Thanks for the changes so far!

How are you thinking to change the default argument? Right now I think we should provide it as users through "wbcli", and change it when we receive the message that it was updated. Perhaps we can also include a configuration file where we keep this file.

In any case I think we should afterwards store this date in the variables yaml file of each briefing and get it from a function within the wbcli library.

I can help with this if we agree is a good idea.

Reply:
Whatever fits the current workflow with `wbcli` best. My only concern is, that everyone using the satellite track predictions should in most cases explicitly be aware about which forecast has been used (so maybe an argument to `wbcli` is a good idea). Maybe it's also a good idea to print the forecast day of the satellite track somewhere on the plot.

Reply:
Thanks! I will work on the changes to the repo then. Making the date as an input argument for `wbcli`. The easiest place to save the track date is the quarto yaml file of each briefing, which can then be used to include the date in each presentation. We could also included the dates in the figures, but would need a change in the interface of the functions that generate the figures, so I would not recommend it in the moment. I will start including them in the yaml file.

Reply:
Actually had an error in my reasoning above, we will need to pass the track date to the figures. I will include it in the figures.